zebra-state/src/sled_state.rs

```diff
@@ -153,6 +153,27 @@ impl FinalizedState {
                 .block
                 .coinbase_height()
                 .expect("valid blocks must have a height");
+
+            if prev_hash == block::Hash([0; 32]) {
+                assert_eq!(
+                    1,
```

Comment:
An empty state contains zero blocks - the genesis block has not been committed yet.

```suggestion
                    0,
```